core/invehicle-digital-twin/src/invehicle_digital_twin_config.rs

```diff
@@ -15,10 +15,7 @@ pub struct Settings {
 
 /// Load the settings.
 pub fn load_settings() -> Settings {
-    let config =
-        Config::builder().add_source(File::new(CONFIG_FILENAME, FileFormat::Yaml)).build().unwrap();
+    let s = utils::load_settings(CONFIG_FILENAME).unwrap();
```

Comment:
Please try to use a meaningful identifier name rather than single character.

Reply:
We were previously using "settings", rather than "s".

Reply:
whoops, sorry, forgot to rename it